templates/contact-us.html

```diff
@@ -21,23 +21,114 @@
+          <p><a target="_blank" href="https://maps.google.com/?q=51.506312,-0.099871&ll=51.506312,-0.099871&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical USA Inc.</p>
+          <p><small>Link Flex<br>2301 W Anderson Ln<br>Austin, TX 78757<br>United States of America</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=30.355390,-97.730311&ll=30.355390,-97.730311&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical USA Inc.</p>
+          <p><small>18 Tremont Street<br>Suite 210<br>Boston, MA 02108<br>United States of America</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=42.358795,-71.059627&ll=42.358795,-71.059627&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical China</p>
+          <p><small>Room 1246, 12F<br>No. 331 North Caoxi Road<br>Shanghai 200031<br>China<br>上海市漕溪北路331号12楼1246室
+              200031</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=31.190677,121.438980&ll=31.190677,121.438980&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical China</p>
+          <p><small>Unit 1118-19, Level 11, China World Office 1<br>1 Jianguomenwai Avenue, Chaoyang District<br>Beijing
+              P.R.C. 100004<br>China<br>北京市朝阳区建国门外大街1号国贸写字楼1座11层1118-19室 100004</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=39.909514,116.458169&ll=39.909514,116.458169&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical Limited Taiwan Branch</p>
+          <p><small>Room D, 46F<br>No. 7, Xin Yi Rd., Sec.5<br>Taipei 101<br>Taiwan</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=25.034010,121.564461&ll=25.034010,121.564461&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical Limited</p>
+          <p><small>One Circular Road<br>Douglas<br>Isle of Man<br>IM1 1SB<br>United Kigdom</small></p>
+          <p><a target="_blank" href="https://maps.google.com/?q=54.15272,-4.487221&ll=54.15272,-4.487221&z=18"><small>View map&nbsp;&rsaquo;</small></a></p>
+        </div>
+        <div class="col-4 p-card" style="border-style: none;">
+          <p class="p-card__title">Canonical limited</p>
```

Comment:
"Limited" should be capitalised here:
```suggestion
          <p class="p-card__title">Canonical Limited</p>
```